warehouse/validations/validate.go

```diff
@@ -251,7 +251,7 @@ func NewValidator(step string, dest *backendconfig.DestinationT) (Validator, err
 		return &loadTable{
 			destination: dest,
 			manager:     operations,
-			table:       Table,
```

Comment:
Unrelated to this PR, but `Table` could be unexported and it could be a `const`.